client/src/context/useAuthContext.tsx

```diff
@@ -56,19 +56,17 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
       .catch((error) => console.error(error));
   }, [history]);
 
-  useEffect(() => {
-    const hasProfile = async () => {
-      await profileGetByUser().then((data: ProfileCreateSuccess) => {
-        if (data.profile) {
-          updateProfileContext(data);
-          history.push('/dashboard');
-        } else if (data.error) {
-          history.push('/edit-profile');
-        }
-      });
-    };
-    hasProfile();
-  }, [history, loggedInUser, updateProfileContext]);
+  // useEffect(() => {
+  //   const hasProfile = async () => {
+  //     await profileGetByUser().then((data: ProfileCreateSuccess) => {
+  //       if (data.profile) {
+  //         updateProfileContext(data);
+  //         history.push('/dashboard');
+  //       }
+  //     });
+  //   };
+  //   hasProfile();
+  // }, [history, loggedInUser, updateProfileContext]);
```

Comment:
Unused code should be removed

Reply:
Its buggy code ithat causing re renders,working on fixing.

Reply:
Since you've mentioned that you are working on fixing this bug, don't forget to make a hotfix for it :)